### ISSUE TITLE
Set antenna offsets

### ISSUE THREAD

**Issue by DavidTMuc** [dawonn/vectornav#77]
Hello,

I have a VN-300 Dual GNSS and wondering if I could set the Antenna positions directly in your code, instead of the VectorNavs control center??

Thanks!!

**Comment by DavidTMuc** [comment]
![image](https://user-images.githubusercontent.com/84916423/129622804-097f8de4-30f8-4d4f-97e8-56d72f6ac9c6.png)
![image](https://user-images.githubusercontent.com/84916423/129622864-013d47af-497c-474e-974b-1ad0759b80f6.png)

Here are the settings in the Control Center as an example.

**Comment by dawonn** [comment]
The ROS2 driver supports this via parameters, but the ROS1 version
doesn't.  You'll have to open up main.cbp and add a little section to set
those parameters, but it should be pretty straightforward, don't be too
afraid. ;)

**Comment by DavidTMuc** [comment]
It finally worked!! In the following the code:
// Set Antenna position
    vec3f antenna_position(1.1, 2.2, 3.3);
    vs.writeGpsAntennaOffset(user_data.antenna_position, false);
    vec3f base_line_position(1.1, 2.2, 3.3);
    vec3f base_line_uncertainity(0.1, 0.2, 0.3);
    vs.writeGpsCompassBaseline(user_data.base_line_position,user_data.base_line_uncertainity, false);
    vs.writeSettings();
    
Now I wonder how I could refer to this in the launch file. Could you give me an advice - especially regarding the node handle, which I have to use to solve this issue, I guess?
 
 Thank you!!

**Comment by dawonn** [comment]
Yep, use the ROS parameter system to read in those parameters from a launch
file. You can see some examples where I have a parameter for the serial
port and baud rate.
